OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/AlertDialogFragment.java

```diff
@@ -29,6 +38,23 @@ public static AlertDialogFragment newInstance(@Nullable String title, @NonNull S
         return fragment;
     }
 
+    /**
+     * Creates and shows a new instance of this dialog fragment with the provided title and message.
+     *
+     * @param title   The title to set.
+     * @param message The message to set.
+     */
+    public static void showDialog(@NonNull FragmentManager manager,
+                                  @Nullable String title, @NonNull String message) {
+        try {
+            newInstance(title, message).show(manager, null);
+        } catch (IllegalStateException e) {
+            // Do nothing, since this is a known issue in the support package
+            // Bug: http://code.google.com/p/android/issues/detail?id=19917
```

Comment:
This is not a bug in the support package, particularly not that one, and the StackOverflow [answer][1] that claims this doesn't seem to make any sense anywhere. Instead, this is the consequence of the combination of not having session management in our tasks (e.g. such as in the MVP framework), the fact that the state can be saved by the system at any point, and the fact that `DialogFragment` doesn't have any `showAllowingStateLoss()` method (even though it does have a corresponding `dismissAllowingStateLoss()` method).

[1]: https://stackoverflow.com/a/10261449